internal/provider/adc/translator/apisixconsumer.go

```diff
@@ -0,0 +1,360 @@
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package translator
+
+import (
+	"fmt"
+	"strconv"
+
+	"github.com/pkg/errors"
+	k8stypes "k8s.io/apimachinery/pkg/types"
+
+	adctypes "github.com/apache/apisix-ingress-controller/api/adc"
+	v2 "github.com/apache/apisix-ingress-controller/api/v2"
+	"github.com/apache/apisix-ingress-controller/internal/controller/label"
+	"github.com/apache/apisix-ingress-controller/internal/provider"
+)
+
+var (
+	_errKeyNotFoundOrInvalid      = errors.New("key \"key\" not found or invalid in secret")
+	_errUsernameNotFoundOrInvalid = errors.New("key \"username\" not found or invalid in secret")
+	_errPasswordNotFoundOrInvalid = errors.New("key \"password\" not found or invalid in secret")
+)
+
+const (
+	_jwtAuthExpDefaultValue = int64(868400)
+
+	_hmacAuthAlgorithmDefaultValue           = "hmac-sha256"
+	_hmacAuthClockSkewDefaultValue           = int64(0)
+	_hmacAuthKeepHeadersDefaultValue         = false
+	_hmacAuthEncodeURIParamsDefaultValue     = true
+	_hmacAuthValidateRequestBodyDefaultValue = false
+	_hmacAuthMaxReqBodyDefaultValue          = int64(524288)
```

Comment:
可以不用显示指定 int64，直接为  untyped，使用时 go 自动适应类型。
```suggestion
	_hmacAuthMaxReqBodyDefaultValue          = 524288
```